code/datums/status_effects/debuffs/drunk.dm

```diff
@@ -188,8 +188,11 @@
 				to_chat(owner, span_warning("You're so tired... but you can't miss that shuttle..."))
 
 			else
-				to_chat(owner, span_warning("Just a quick nap..."))
-				owner.Sleeping(90 SECONDS)
+				blackout()
```

Comment:
1. If no ghosts take it it should default to what it used to do (long sleep)
2. This should also have a different effect if rolled when already "blackout"